staging/src/k8s.io/apiserver/pkg/server/routes/version.go

```diff
@@ -39,11 +39,11 @@ func (v Version) Install(c *restful.Container) {
 	// Set up a service to return the git code version.
 	versionWS := new(restful.WebService)
 	versionWS.Path("/version")
-	versionWS.Doc("git code version from which this is built")
+	versionWS.Doc("get the version information for this server.")
 	versionWS.Route(
 		versionWS.GET("/").To(v.handleVersion).
-			Doc("get the code version").
-			Operation("getCodeVersion").
```

Comment:
Why is this change needed?  I don't know enough about openapi offhand to know if this is semantically important, but it smells semantically important.  I wouldn't expect it to change unless we must change it.